src/App/Color.cpp

```diff
@@ -89,6 +89,24 @@ uint32_t Color::getPackedValue() const
     // clang-format on
 }
 
+void Color::setPackedRGB(uint32_t rgb)
+{
+    // clang-format off
+    this->set(static_cast<float>((rgb >> 24) & 0xff) / 255.0F,
+              static_cast<float>((rgb >> 16) & 0xff) / 255.0F,
+              static_cast<float>((rgb >>  8) & 0xff) / 255.0F);
+    // clang-format on
+}
+
+uint32_t Color::getPackedRGB() const
+{
+    // clang-format off
+    return (static_cast<uint32_t>(r * 255.0F + 0.5F) << 24 |
```

Comment:
The linter is a bit unhappy with this -- obviously the potential for incorrect rounding here is basically inconsequential, but is there a good reason *not* to use `lround`?

Reply:
Replaced the code with std::lround